### ISSUE TITLE
False positive "unnecessary parenthesis" on calling anonymous function

### ISSUE THREAD

**Issue by velosipedist** [kalessil/phpinspectionsea#742]
This will raise "unnecessary parenthesis" inspection but without braces PHP will show PARSE_ERROR

```php
(function ($foo) {
            $this->privateField = $foo;
        })->call($entity, $privateValue);
```

**Comment by kalessil** [comment]
Thank you for reporting @velosipedist, this has been fixed already in #735.
